contrib/win32/win32compat/shell-host.c

```diff
@@ -980,6 +1025,20 @@ ConsoleEventProc(HWINEVENTHOOK hWinEventHook,
 }
 
 DWORD 
+GetCurrentTerminalType(void)
+{
```

Comment:
During my testing, all clients sent a environment TERM to the server:
- Putty send 'xterm'
- Bitvise send 'xterm'
- TeraTerm send 'vt100'
- OpenSSH-ubuntu send 'xterm'

Only OpenSSH-Win32 do not send environment TERM.